src/app/[activityId]/page.tsx

```diff
@@ -35,27 +33,26 @@ export default function ActivityDetail() {
   const isTablet = useMediaQuery('tablet');
   const isMobile = useMediaQuery('mobile');
 
+  if (!activityId || Array.isArray(activityId)) notFound();
+  const id = Number(activityId);
+  if (isNaN(id)) notFound();
```

Comment:
```suggestion
  if (!activityId || isNaN(Number(id)) || Array.isArray(activityId)) notFound();
```
💬 이렇게 한 문장으로 통일해도 되지 않나요? 다만 그냥 `isNaN(Number(id))`만 써도 똑같지 않을까 싶네요